conf/modules.config

```diff
@@ -42,6 +42,14 @@ process {
         ]
     }
 
+    withName: SAMTOOLS_DEPTH{
+        ext.args = { "-aa" }
+    }
+
+    withName: SAMTOOLS_SORT{
+        ext.prefix  = { "${meta.id}_sorted" }
+    }
+
     withName: KRAKEN2_KRAKEN2 {
         ext.args = { "--report-zero-counts --use-names --memory-mapping" }
```

Comment:
`--memory-mapping` avoids loading the database in memory. So with `--memory-mapping` it should use less memory but run slower. The [https://github.com/sanger-tol/cobiontcheck/tree/develop](https://github.com/sanger-tol/cobiontcheck/tree/develop) version of the pipeline doesn't use memory mapping when running Kraken2. I guess whether to use memory mapping or not is still up for discussion but to replicate what the [https://github.com/sanger-tol/cobiontcheck/tree/develop](https://github.com/sanger-tol/cobiontcheck/tree/develop) pipeline version does, `--memory-mapping` should be removed from here